deploy/09_set_minimum_stake.ts

```diff
@@ -0,0 +1,23 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { getNamedAccounts, deployments } = hre
+  const { execute } = deployments
+  const { log } = deployments
+
+  const { deployer } = await getNamedAccounts()
+
+  const minStakeAmount = "39999999999999999999999"
+  await execute("TokenStaking", { from: deployer }, "setMinimumStakeAmount")
```

Comment:
When testing the deployment with the modified values, I got `Error: expected 1 arguments for method "setMinimumStakeAmount", got 0` error.
Full logs: https://github.com/threshold-network/solidity-contracts/runs/7782586514?check_suite_focus=true

Reply:
c5bdf3d

Reply:
After the fix the deployment worked: https://github.com/threshold-network/solidity-contracts/runs/7801852357?check_suite_focus=true.